pulp_ansible/app/migrations/0049_ansiblenamespacemetadata_avatar_url.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.18 on 2023-03-17 13:19
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ansible', '0048_collectionversionmark'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='ansiblenamespacemetadata',
+            name='avatar_url',
+            field=models.URLField(blank=True, max_length=256),
+        ),
```

Comment:
We haven't released Namespace Metadata yet. I think we can just add this to the original migration. It should only break dev envs.

Reply:
👍